Gemfile

```diff
@@ -41,7 +41,9 @@ end
 
 # ActiveSupport
 gem 'dalli', '>= 2.2.1'
-
+group :test do
+  gem 'listen', '~>1.0'
```

Comment:
Actually features of listen gem used in this api were there in 1.0. So I thought that we can work with stable old verson.